service/compute/commands.go

```diff
@@ -9,6 +9,63 @@ import (
 	"github.com/databricks/databricks-sdk-go/useragent"
 )
 
+type CommandExecutorV2 struct {
+	clustersAPI  *ClustersAPI
+	executionAPI *CommandExecutionAPI
+	language     Language
+	clusterID    string
+	contextID    string
+}
```

Comment:
Testing this would be painful, consider turning this into an interface like `CommandExecutor` and then provide an implementation instead. Then it'll be much easier to drop in a mock when testing downstream usage.

Reply:
yep. that's the plan. TF has plenty of resources that call into this api